content/shadow-dom/sections/removing-shadow-dom-styles.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Removing Shadow DOM styles
+---
+
+### Removing Shadow DOM styles
+
+In Atom `1.13` the Shadow DOM got removed from text editors. For more background on the reasoning, check out the [Pull Request](https://github.com/atom/atom/pull/12903). In this guide you will learn how to migrate your theme or package.
```

Comment:
Maybe _"[...] check out the [pull request](https://github.com/atom/atom/pull/12903) where it was got removed. [...]"_ or something along those lines?